client/my-sites/checkout/composite-checkout/lib/translate-cart.ts

```diff
@@ -462,7 +462,15 @@ export function getSublabel( serverCartItem: ResponseCartProduct ): i18nCalypso.
 		}
 	}
 
-	return isRenewalItem ? translate( 'Renewal' ) : '';
+	if ( ! isRenewalItem && serverCartItem.months_per_bill_period === 1 ) {
+		return translate( 'Monthly subscription' );
+	}
```

Comment:
Without this then all the components dependent on the existing of a sublabel won't be rendered (including the discount callouts). This also is helpful to highlight a monthly subscription since our usual billing intervals are annual.